config/jobs/etcd-druid/etcd-druid-api-unit-tests.yaml

```diff
@@ -0,0 +1,64 @@
+presubmits:
+  gardener/etcd-druid:
+    - name: pull-etcd-druid-api-unit
+      cluster: gardener-prow-build
+      always_run: true
+      optional: true
+      skip_branches:
+        - hotfix-v\d+.\d+ # don't run on release branches for now (add a job per branch later)
```

Comment:
I don't understand the `add a job per branch later` part. You specified `fork-per-release: "true"` and AFAIK this should create jobs automatically per release branch. At least this is the behaviour for gardener/gardener. Do you really need to maintain jobs per hotfix branches manually?

Reply:
Ah this was because it was copied over from `etcd-druid-unit-tests.yaml`